airflow/cli/commands/pool_command.py

```diff
@@ -21,61 +21,74 @@
 import sys
 from json import JSONDecodeError
 
-from tabulate import tabulate
-
 from airflow.api.client import get_current_api_client
+from airflow.cli.simple_table import AirflowConsole
 from airflow.utils import cli as cli_utils
+from airflow.utils.cli import suppress_logs_and_warning
 
 
-def _tabulate_pools(pools, tablefmt="fancy_grid"):
-    return "\n%s" % tabulate(pools, ['Pool', 'Slots', 'Description'], tablefmt=tablefmt)
+def _show_pools(pools, output):
+    AirflowConsole().print_as(
+        data=pools,
+        output=output,
+        mapper=lambda x: {
+            "pool": x[0],
+            "slots": x[1],
+            "description": x[2],
+        },
+    )
 
 
+@suppress_logs_and_warning()
 def pool_list(args):
     """Displays info of all the pools"""
     api_client = get_current_api_client()
     pools = api_client.get_pools()
-    print(_tabulate_pools(pools=pools, tablefmt=args.output))
+    _show_pools(pools=pools, output=args.output)
 
 
+@suppress_logs_and_warning()
 def pool_get(args):
     """Displays pool info by a given name"""
     api_client = get_current_api_client()
     pools = [api_client.get_pool(name=args.pool)]
-    print(_tabulate_pools(pools=pools, tablefmt=args.output))
+    _show_pools(pools=pools, output=args.output)
 
 
 @cli_utils.action_logging
+@suppress_logs_and_warning()
 def pool_set(args):
     """Creates new pool with a given name and slots"""
     api_client = get_current_api_client()
     pools = [api_client.create_pool(name=args.pool, slots=args.slots, description=args.description)]
-    print(_tabulate_pools(pools=pools, tablefmt=args.output))
+    _show_pools(pools=pools, output=args.output)
```

Comment:
Same here, most of clis I know would show "Pool created" instead of showing info. We can consider a verbose flag.